### PR TITLE
Add Dockerfile to build jenkins image and fix startup script

### PR DESCRIPTION
**Pull request by kbaladurin**

### CODE REVIEW

**Comment by kbaladurin** [comment]
@gbalykov @alpencolt PTAL

File: tools/jenkins-run.sh [hunk 1]
```diff
@@ -43,7 +45,7 @@ if [[ ${N_TOTAL_CONTAINER} -ne 0 ]]; then
     exit
 fi
 
-DOCKER_OPT=()
+DOCKER_OPT=("-v" "/var/run/docker.sock:/var/run/docker.sock")
```
Comment (gbalykov) [inline tools/jenkins-run.sh:48]: Why do we need this?

File: tools/jenkins-run.sh [hunk 2]
```diff
@@ -52,4 +54,4 @@ else
     DOCKER_OPT+=("-v" "${__home}:/var/jenkins_home")
 fi
 
-docker run -d --name ${__name} -p 8080:8080 -p 50000:50000 ${DOCKER_OPT[@]} jenkins
+docker run -d --group-add $(getent group docker | awk -F: '{print $3}') --name ${__name} -p 8080:8080 -p 50000:50000 ${DOCKER_OPT[@]} jenkins
```
Comment (gbalykov) [inline tools/jenkins-run.sh:57]: Why do we need group-add?

File: tools/jenkins-run.sh (hunk 1 above)
Comment (kbaladurin) [reply tools/jenkins-run.sh:48]: We need this to run docker from our container

File: tools/jenkins-run.sh (hunk 2 above)
Comment (kbaladurin) [reply tools/jenkins-run.sh:57]: We need  to add docker guid to user to allow him to use `/var/run/docker.sock`. The problem is that docker guid inside container can be not equal to docker guid on host so we add host's docker guid via ` --group-add`.

**Review by gbalykov — APPROVED** [review]
LGTM

**Pull request merged**